data/LogicHelpers.json

```diff
@@ -99,53 +113,50 @@
+    "can_kill_tailsparan": "has_soul(Tailpasaran) and ((is_child and ((Kokiri_Sword and (Magic_Meter or can_take_damage)) or Sticks or Boomerang or can_use(Megaton_Hammer))) or (is_adult and (Magic_Meter or can_take_damage or Megaton_Hammer)))",
+    "can_kill_stinger": "has_soul(Stinger) and (is_adult or Kokiri_Sword or Sticks or Slingshot or has_explosives or can_use(Megaton_Hammer))",
+    "can_kill_tektite": "has_soul(Tektite) and (is_adult or Kokiri_Sword or Sticks or Slingshot or has_explosives or can_use(Megaton_Hammer))",
     "can_kill_skulltula": "has_soul(Skulltula) and (is_adult or can_child_damage)",
-    "can_kill_wolfos": "has_soul(Wolfos) and (is_adult or Slingshot or Sticks or Kokiri_Sword or can_use(Dins_Fire))",
-    "can_kill_redead": "has_soul(Redead_and_Gibdo) and (is_adult or Sticks or Kokiri_Sword or can_use(Dins_Fire))",
-    "can_kill_keese": "has_soul(Keese) and (is_adult or Kokiri_Sword or Sticks or Slingshot or Boomerang or (Nuts and has_explosives))",
-    "can_kill_keese_stationary": "has_soul(Keese) and (is_adult or Kokiri_Sword or Sticks or Slingshot or Boomerang or has_explosives)",
-    "can_kill_scrub": "has_soul(Deku_Scrub) and (is_adult or Kokiri_Sword or Sticks or Slingshot or Boomerang or can_use(Dins_Fire) or (Nuts and has_explosives))",
-    "can_kill_blue_bubble": "has_soul(Bubble) and (Bombs or (is_adult and (Nuts or Hylian_Shield or Mirror_Shield or Hookshot or Megaton_Hammer or Bow)) or (is_child and (Boomerang or Nuts or Deku_Shield or Hylian_Shield) and (Sticks or Kokiri_Sword or Slingshot)))",
-    "can_kill_green_bubble": "has_soul(Bubble) and (is_adult or Bombs or Sticks or Kokiri_Sword or Slingshot)",
-    "can_kill_floormaster": "has_soul(Floormaster) and (is_adult or Slingshot or Sticks or Kokiri_Sword or (Bombs and Nuts) or has_bombchus or can_use(Dins_Fire))",
+    "can_kill_wolfos": "has_soul(Wolfos) and (is_adult or Slingshot or Sticks or Kokiri_Sword or can_use(Dins_Fire) or can_use(Megaton_Hammer))",
+    "can_kill_redead": "has_soul(Redead_and_Gibdo) and (is_adult or Sticks or Kokiri_Sword or can_use(Dins_Fire) or can_use(Megaton_Hammer))",
+    "can_kill_keese": "has_soul(Keese) and (is_adult or Kokiri_Sword or Sticks or Slingshot or Boomerang or (Nuts and has_explosives) or can_use(Megaton_Hammer))",
+    "can_kill_keese_stationary": "has_soul(Keese) and (is_adult or Kokiri_Sword or Sticks or Slingshot or Boomerang or has_explosives or can_use(Megaton_Hammer))",
+    "can_kill_scrub": "has_soul(Deku_Scrub) and (is_adult or Kokiri_Sword or Sticks or Slingshot or Boomerang or can_use(Dins_Fire) or (Nuts and has_explosives) or can_use(Megaton_Hammer))",
+    "can_kill_blue_bubble": "has_soul(Bubble) and (Bombs or (is_adult and (Nuts or Hylian_Shield or Mirror_Shield or Hookshot or Megaton_Hammer or Bow)) or (is_child and (child_hammer or ( (Boomerang or Nuts or Deku_Shield or Hylian_Shield) and (Sticks or Kokiri_Sword or Slingshot) )) ))",
```

Comment:
so it seems like you made a child_hammer helper. It seems like most of the other can_use(Megaton_Hammer) added here are also meant for child hammer. if that's the case, please use the child_hammer helper instead of can_use(Megaton_Hammer)